internal/operators/api/api.go

```diff
@@ -38,7 +38,7 @@ type Operator interface {
 	// ValidateCluster verifies whether this operator is valid for given cluster
 	ValidateCluster(ctx context.Context, cluster *common.Cluster) (ValidationResult, error)
 	// ValidateHost verifies whether this operator is valid for given host
-	ValidateHost(ctx context.Context, cluster *common.Cluster, hosts *models.Host) (ValidationResult, error)
+	ValidateHost(ctx context.Context, cluster *common.Cluster, hosts *models.Host, interOperatorRequirements *models.ClusterHostRequirementsDetails) (ValidationResult, error)
```

Comment:
Maybe name this just `additionalOperatorRequirements`?
I could see this being useful for other runtime stuff that might come up.